devicetypes/smartthings/ct100-thermostat.src/ct100-thermostat.groovy

```diff
@@ -20,9 +20,9 @@ metadata {
 		command "raiseCoolSetpoint"
 		command "poll"
 
-		fingerprint deviceId: "0x08", inClusters: "0x43,0x40,0x44,0x31,0x80,0x85,0x60"
-		fingerprint mfr:"0098", prod:"6401", model:"0107", deviceJoinName: "2Gig CT100 Programmable Thermostat"
-		fingerprint mfr:"0098", prod:"6501", model:"000C", deviceJoinName: "Radio Thermostat CT101"
+		fingerprint deviceId: "0x08", inClusters: "0x43,0x40,0x44,0x31,0x80,0x85,0x60", deviceJoinName: "Thermostat"
+		fingerprint mfr:"0098", prod:"6401", model:"0107", deviceJoinName: "2Gig Thermostat" //2Gig CT100 Programmable Thermostat
+		fingerprint mfr:"0098", prod:"6501", model:"000C", deviceJoinName: "Thermostat" //Radio Thermostat CT101
```

Comment:
Why doesn't this have a manufacturer?

Reply:
I guess I've thought that Radio doesn't mean brand but, well, indicates being wireless... I don't know, back then it seemed to be reasonable...

Reply:
I think you're right. Sorry, I forgot we had discussed this before. I think it should be left as just "Thermostat".